Runner/CustomTemplateFactory.py

```
@@ -0,0 +1,170 @@
+import json
+"""
+This module responsiblilty is for reading and setting elements of the json templates
+"""
+
+
+def set_template_name(template, pool_id):
+    try:
+        template["parameters"]["poolName"]["defaultValue"] = pool_id
+    except KeyError:
+        pass
+    try:
+        template["parameters"]["poolId"]["defaultValue"] = pool_id
+    except KeyError:
+        pass
+    try:
+        template["parameters"]["poolId"]["value"] = pool_id
+    except KeyError:
+        pass
+
+
+def set_parameter_name(template, job_id):
+    try:
+        template["jobName"]["value"] = job_id
+    except KeyError:
+        pass
+    try:
+        template["jobId"]["value"] = job_id
+    except KeyError:
+        pass
```

🔔 same statement repeated `try-except` many times, i.e. I think copy paste error? Why not both `statement` under 1 try block? Also, Redundant `exception` handle.

---

Think this is to make sure if the first one fail the second actually still runs. Also those try except are needed they if ignore the exception which is what is wanted here

---

I've changed this up